NASA-PDS/registry-mgr
# Removed "load-data" command

## 🗒️ Summary
* Moved LDD / schema update and data load code to commons project to be used by Harvest.
* Removed "load-data" command. Now Harvest tool loads data directly.

## ⚙️ Test Data and/or Report
* Run registry-manager without any parameters. Check that help screen doesn't list "load-data" command.
* Run `registry-manager load-data` command. You should get an error message `Invalid command: load-data`.

## ♻️ Related Issues
https://github.com/NASA-PDS/harvest/issues/78
